manifests/0000_70_cluster-network-operator_01_egr_crd.yaml

```diff
@@ -0,0 +1,128 @@
+# This file is automatically generated. DO NOT EDIT
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.0
+    include.release.openshift.io/ibm-cloud-managed: "true"
```

Comment:
What is this annotation for?

Reply:
magic. well, not really. part of https://github.com/openshift/cluster-version-operator/pull/404